CMakeLists.txt

```diff
@@ -466,6 +468,16 @@ if(KTX_FEATURE_JNI)
     add_subdirectory(interface/java_binding)
 endif()
 
+# This is put here so CMake doesn't warn that "PYTHON" is unused if KTX_FEATURE_PY=OFF
+# PYTHON_EXECUTABLE is resolved in python_binding/CMakeLists.txt
```

Comment:
Where is PYTHON set?

Can we put this as an `else()` to the `if(KTX_FEATURE_PY)`?

Reply:
I removed that snippet - seems like it's unnecessary now.